### PR TITLE
init the simplest post list by project

### PR DESCRIPTION
**Pull request by bopoda**

### CODE REVIEW

File: src/AppBundle/Controller/PostController.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    public function onlPostsByProjectAction($project)
+    {
+        $onlPosts = $this->getDoctrine()->getRepository('AppBundle:OnlinerPosts')->findBy([
+            'project' => $project
+        ], [], 20);
+
+        return $this->render('onlPost/list.html.twig', [
+            'onlPosts' => $onlPosts,
+            'project' => $project,
+        ]);
```
Comment (stan220) [inline src/AppBundle/Controller/PostController.php]: you can use compact('onlPosts', 'project'); for return array

Comment (bopoda) [inline src/AppBundle/Controller/PostController.php]: @stan220 hey, what do you mean?

Comment (stan220) [inline src/AppBundle/Controller/PostController.php]: learn http://php.net/manual/ru/function.compact.php

File: app/Resources/views/onlPost/parts/part-post-itemdiv.html.twig [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    </div>
+    <div class="brief">
+        {% if onlPost.onlImage %}
+            <p><a href="{{ path('onlPostView', {'postId': onlPost.id}) }}"><img src="{{ onlPost.onlImage }}" alt="{{ onlPost.onlCaption }}"/></a></p>
+        {% endif %}
+
+        {{ onlPost.onlShortText|raw }}
+    </div>
+    <div class="publish">
+        Комментариев: <span class="green">{{ onlPost.onlCountComments}}</span>
```
Comment (bopoda) [inline app/Resources/views/onlPost/parts/part-post-itemdiv.html.twig]: thank you, fixed

**Pull request merged**